src/Dapper.FluentMap/Utils/FluentMapConfigurationExtensions.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Dapper.FluentMap.Configuration;
+using Dapper.FluentMap.Mapping;
+
+namespace Dapper.FluentMap.Utils
+{
+    /// <summary>
+    /// Extension methods for <see cref="FluentMapConfiguration"/>
+    /// </summary>
+    public static class FluentMapConfigurationExtensions
+    {
+        /// <summary>
+        /// Finds all types, from provided assemblies, implementing <see cref="EntityMap{TEntity}"/>
+        /// and applies them to <see cref="FluentMapConfiguration"/>,
+        /// by calling <see cref="FluentMapConfiguration.AddMap{TEntity}"/> and passing an instance of found type.
+        /// </summary>
+        /// <param name="configuration">The <see cref="FluentMapConfiguration"/> instance</param>
+        /// <param name="assemblies">Assemblies</param>
+        /// <exception cref="ArgumentNullException"></exception>
+        public static void ApplyMapsFromAssemblies(this FluentMapConfiguration configuration,
+            params Assembly[] assemblies)
```

Comment:
```suggestion
        public static void ApplyMapsFromAssemblies(this FluentMapConfiguration configuration,  params Assembly[] assemblies)
```

Can you put the parameters on a single line?